moto/elastictranscoder/responses.py

```diff
@@ -0,0 +1,128 @@
+# Only mock response without implementation of logic in backend
+from moto.core.responses import BaseResponse
+from .models import elastictranscoder_backends
+import json
+
+
+DEFAULT_REGION_NAME = 'us-east-1'
+
+
+class ElastictranscoderResponse(BaseResponse):
+    SERVICE_NAME = 'elastictranscoder'
+
+    @property
+    def elastictranscoder_backend(self):
+        return elastictranscoder_backends[self.region]
+
+    @classmethod
+    def create_preset(clazz, request, full_url, headers):
+        headers['content-type'] = 'application/json'
+        resp = json.loads(CREATE_PRESET_RESP)
+        return 201, headers, json.dumps(resp)
+
+    @classmethod
+    def create_job(clazz, request, full_url, headers):
+        print(request.__dict__)
+        headers['content-type'] = 'application/json'
+        resp = json.loads(CREATE_JOB_RESP)
+        return 201, headers, json.dumps(resp)
+
+    @classmethod
+    def check_job(clazz, request, full_url, headers):
+        headers['content-type'] = 'application/json'
+        resp = json.loads(CHECK_JOB_RESP)
+        return 201, headers, json.dumps(resp)
+
+    @classmethod
+    def delete_preset(clazz, request, full_url, headers):
```

Comment:
PEP8: `Always use cls for the first argument to class methods.` :D, makes sense though as then its consistent.